DOCPLAN-134: Correct product naming (#109665)
openshift/openshift-docs: virt/creating_vms_advanced/creating_vms_advanced_web/virt-creating-vms-from-rh-images-overview.adoc

:_mod-docs-content-type: ASSEMBLY
[id="virt-creating-vms-from-rh-images-overview"]

> = Creating virtual machines from Red Hat Enterprise Linux (RHEL) images

Reviewer: ```suggestion
= Creating virtual machines from {op-system-base-full} images
```

include::_attributes/common-attributes.adoc[]
:context: virt-creating-vms-from-rh-images-overview

toc::[]

[role="_abstract"]

> {op-system-base-full} images are published as container disks in a secure registry. The Containerized Data Importer (CDI) polls and imports the container disks into your cluster and stores them in the `openshift-virtualization-os-images` project as snapshots or persistent volume claims (PVCs). You can optionally use a custom namespace for golden images.

Reviewer 2: ```suggestion
{op-system-base} images are golden images. They are published as container disks in a secure registry. The Containerized Data Importer (CDI) polls and imports the container disks into your cluster and stores them in the `openshift-virtualization-os-images` project as snapshots or persistent volume claims (PVCs). You can optionally use a custom namespace for golden images.
```

> Red Hat Enterprise Linux (RHEL) images are published as container disks in a secure registry. The Containerized Data Importer (CDI) polls and imports the container disks into your cluster and stores them in the `openshift-virtualization-os-images` project as snapshots or persistent volume claims (PVCs). You can optionally use a custom namespace for golden images.

Reviewer: ```suggestion
{op-system-base-full} images are published as container disks in a secure registry. The Containerized Data Importer (CDI) polls and imports the container disks into your cluster and stores them in the `openshift-virtualization-os-images` project as snapshots or persistent volume claims (PVCs). You can optionally use a custom namespace for golden images.
```

> {op-system-base-full} images are automatically updated. You can disable and re-enable automatic updates for these images. See xref:../../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[Managing Red Hat boot source updates].

Reviewer 2: ```suggestion
{op-system-base} images are automatically updated. You can disable and re-enable automatic updates for these images. For more information, see "Additional resources".
```

> Red Hat Enterprise Linux (RHEL) images are automatically updated. You can disable and re-enable automatic updates for these images. See xref:../../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[Managing Red Hat boot source updates].

Reviewer: ```suggestion
{op-system-base-full} images are automatically updated. You can disable and re-enable automatic updates for these images. See xref:../../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[Managing Red Hat boot source updates].
```

> Cluster administrators can enable automatic subscription for {op-system-base-full} virtual machines in the {VirtProductName} xref:../../../virt/about_virt/about-virt.adoc#about-virt[web console].

Reviewer 2: ```suggestion
Cluster administrators can enable automatic subscription for {op-system-base} virtual machines in the  {product-title} web console.
```

:_mod-docs-content-type: ASSEMBLY
[id="virt-creating-vms-from-rh-images-overview"]
= Creating virtual machines from {op-system-base-full} images
include::_attributes/common-attributes.adoc[]

> :context: virt-creating-vms-from-rh-images-overview

Reviewer 3: If you plan to use an attribute in the title you probably need to reorder this metadata to put the attributes include before the title, i.e.

````
:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="virt-creating-vms-from-rh-images-overview"]
= Creating virtual machines from {op-system-base-full} images
:context: virt-creating-vms-from-rh-images-overview
````

I don't know enough to know if `{op-system-base-full}` is the right name, but if the name is being updated here, the title of the assembly probably needs to be updated in the topicmap(s) as well?

Author: Hi @abrennan89 -- thanks for the review! I've reordered the metadata so the `include::` line comes before the title.

Regarding the topicmap — I checked and `{op-system-base-full}` resolves to two different values depending on the distro: `Red Hat Enterprise Linux (RHEL)` and `Fedora`. Since the attribute doesn't render in the topicmap, could you advise on what the `Name:` field should be updated to, or whether it should be left as-is?

Reviewer 3: The name in the topicmap should always match whatever title is used in the assembly, so if that's the case I wouldn't use the attribute in the title.

toc::[]

[role="_abstract"]
{op-system-base} images are golden images. They are published as container disks in a secure registry. The Containerized Data Importer (CDI) polls and imports the container disks into your cluster and stores them in the `openshift-virtualization-os-images` project as snapshots or persistent volume claims (PVCs). You can optionally use a custom namespace for golden images.

{op-system-base} images are automatically updated. You can disable and re-enable automatic updates for these images. For more information, see "Additional resources".

> CCluster administrators can enable automatic subscription for {op-system-base} virtual machines in the  {product-title} web console.

Reviewer 2: ```suggestion
Cluster administrators can enable automatic subscription for {op-system-base} virtual machines in the  {product-title} web console.
```

toc::[]

[role="_abstract"]
{op-system-base} images are golden images. They are published as container disks in a secure registry. The Containerized Data Importer (CDI) polls and imports the container disks into your cluster and stores them in the `openshift-virtualization-os-images` project as snapshots or persistent volume claims (PVCs). You can optionally use a custom namespace for golden images.

{op-system-base} images are automatically updated. You can disable and re-enable automatic updates for these images. For more information, see "Additional resources".

> Cluster administrators can enable automatic subscription for {op-system-base} virtual machines in the {product-title} web console.chk

Reviewer 2: ```suggestion
Cluster administrators can enable automatic subscription for {op-system-base} virtual machines in the {product-title} web console.
```

include::modules/virt-golden-images-namespace-web.adoc[leveloffset=+1]

include::modules/virt-golden-images-namespace-cli.adoc[leveloffset=+1]

[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources

* xref:../../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[Managing Red{nbsp}Hat boot source updates]

> 

Reviewer 2: Please remove the blank lines between the xrefs

include::modules/virt-golden-images-namespace-web.adoc[leveloffset=+1]

include::modules/virt-golden-images-namespace-cli.adoc[leveloffset=+1]

> [role="_additional-resources"]

Reviewer 2: ```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
```

> == Additional resources

Reviewer 2: ```suggestion

```

> * xref:../../../virt/about_virt/about-virt.adoc#about-virt[Web console]

Reviewer 2: This page has no reference to the web console. Can you either remove this link or update it to point to the correct page?

> * xref:../../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[Managing Red Hat boot source updates]

Reviewer 2: ```suggestion
* xref:../../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[Managing Red{nbsp}Hat boot source updates]
```
